w3af/core/data/parsers/doc/url.py

```diff
@@ -214,6 +220,7 @@ def __init__(self, data, encoding=DEFAULT_ENCODING):
         self._netloc = None
         self._path = None
         self._params = None
+        self._hostname = None
```

Comment:
You added this attribute here and to `__slots__` and then never used it? It is confusing to see `self._hostname` and `self.hostname`